force-app/main/slettedeVirksomheter/classes/KafkaSlettedeVirksomheterHandler.cls

```diff
@@ -0,0 +1,26 @@
+public without sharing class KafkaSlettedeVirksomheterHandler implements IKafkaMessageConsumer {
+	public void processMessages( List<KafkaMessage__c> messages ) {
+		convertKafka( messages );
```

Comment:
```suggestion
		Set<Id> orgNrs = convertKafka( messages );
		accountsToUpdate(orgNrs);
```

ville heller ha returnert første funksjonen, for da kan du skrive en test spesifikt for convertKafka, selv om den ikke gjør så mye